templates/resources/search_deployment.md.tmpl

```diff
@@ -1,7 +1,7 @@
 ---
 layout: "mongodbatlas"
 page_title: "MongoDB Atlas: {{.Name}}"
-subcategory: "docs_{{ .Name }}_{{.Type | lower}}"
+sidebar_current: "docs-{{ .ProviderShortName }}-{{ .Type | lower }}-{{ $name := split .Name "_" }}{{ index $name 1 }}-{{ index $name 2 }}"
```

Comment:
Could you also update the templates https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/templates/data-source.md.tmpl and https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/templates/resources.md.tmpl?

Reply:
Good point, I updated the 2 templates. I also changed a bit how it works so it's more generic and would work for all resources even if the resource name has a different amount of words than search_deployment